### PR TITLE
feat(docs): PostgreSQL guide

### LINKED ISSUE #80: Add Guide for SQL
SQL needs a guide similar to the GO Guide.

### PR DESCRIPTION
**Pull request by Mr-Leshiy**
# Description

Added PostgreSQL builder guide.

## Related Issue(s)

Closes #80

### CODE REVIEW

**Comment by Mr-Leshiy** [comment]
deployed docs: https://input-output-hk.github.io/catalyst-ci/branch/feat_sql_guides/guides/languages/postgresql/

File: docs/src/guides/languages/postgresql.md [hunk 1]
```diff
@@ -9,4 +9,301 @@ tags:
...
+```
+
+There are 4 possible options how to run:
+
+* If `DB_HOST` env var established to `localhost`,
+PostgreSQL server runs as a part of the `example` service,
+otherwise will relies on remote PostgreSQL server connection
+(as an example already defined `postgres` service).
+* `INIT_AND_DROP_DB` env var defines to run initial initialization of the db with the clean state or not
+(could be omitted).
```
Comment (stevenj) [inline docs/src/guides/languages/postgresql.md:179]: Use the word `optional` instead.  But also put a description of what happens when the option is NOT specified.  Does it default, does it cause the container to behave differently? 

Same comment applies to each option below.

**Review by stevenj — CHANGES_REQUESTED** [review]

File: docs/src/guides/languages/postgresql.md (hunk 1 above)
Comment (Mr-Leshiy) [reply docs/src/guides/languages/postgresql.md]: fixed

**Review by stevenj — APPROVED** [review]
LGTM

**Pull request merged**